src/content/docs/setup/configuration/aem-assets-configuration.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: AEM Assets integration
+description: Learn how to connect your Edge Delivery Services storefront to your AEM Assets integration.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Vocabulary from '@components/Vocabulary.astro';
+import { Tabs, TabItem, Aside } from '@astrojs/starlight/components';
+
+The AEM Assets integration allows your storefront to display product images that are managed in AEM Assets instead of traditional Commerce-hosted images. This integration provides enhanced image management capabilities, including advanced optimization, cropping, and delivery through Adobe's Content Delivery Network (CDN).
+
+## Boilerplate update required
+
+Only one update is required: set `"commerce-assets-enabled": true` in your `config.json` file, renamed from `/demo-config.json`. See [Storefront configuration](/setup/configuration/commerce-configuration/) for the full configuration model.
```

Comment:
We should not be pushing people to `demo-config.json` or `config.json` in repo. Just link to the configuration page.

```suggestion
Only one update is required: set `"commerce-assets-enabled": true` in your [Storefront configuration](/setup/configuration/commerce-configuration/).
```